mirror/fileRepository/mirror.sh

```diff
@@ -33,30 +33,25 @@ mirror_kubeupv2(){
         if [[ ! -d $TMPDIR/$platform/amd64 ]]; then
         mkdir -p $TMPDIR/$platform/amd64
         fi
-        if [[ ! -d $TMPDIR/images ]]; then
-        mkdir -p $TMPDIR/images
-        fi
+        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops -O $TMPDIR/$platform/amd64/kops
+        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops.sha1 -O $TMPDIR/$platform/amd64/kops.sha1
     done
 
 
-    for i in nodeup utils.tar.gz kops
-    do
-    for platform in linux darwin
+    for file in nodeup utils.tar.gz
     do
-        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/$i -O $TMPDIR/$platform/amd64/$i
-        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/$i.sha1 -O $TMPDIR/$platform/amd64/$i.sha1
-    done
+        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/linux/amd64/$file -O $TMPDIR/linux/amd64/$file
```

Comment:
We still tend to mirror platform for mac os x especially the`kops` client

Reply:
I noticed that is why [I moved it up](https://github.com/nwcdlabs/kops-cn/blob/e7c72cde2a449da950108d465e9c67b3ac61eb6f/mirror/fileRepository/mirror.sh#L36-L37)

```
        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops -O $TMPDIR/$platform/amd64/kops
        wget -c https://kubeupv2.s3.amazonaws.com/kops/$KOPS_VERSION/$platform/amd64/kops.sha1 -O $TMPDIR/$platform/amd64/kops.sha1
```